src/TestInfrastructure/Framework/ConfigurableHost.cs

```diff
@@ -106,8 +104,6 @@ public void SetActiveSection(ISectionController section)
             this.VisualStateManager.SyncCommandFromActiveSection();
         }
 
-        public ISet<Language> SupportedPluginLanguages { get; } = new HashSet<Language>();
-
```

Comment:
The property is no longer used anywhere so I removed it from the interfaces and implementations.